python/hail/typecheck/check.py

```diff
@@ -115,7 +123,7 @@ def oneof(*args):
 
 
 def nullable(t):
-    return oneof(t, NoneType)
+    return oneof(t, type(None))
```

Comment:
`oneof` takes a varargs of either types or checkers. If it finds a type, it calls `only` on the type. I think it's just a little clearer to use the defined `none` checker here instead of using the type again.